client/state/gutenberg-fse-settings/init.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+export const makeQueryKey = ( siteId: string ): string[] => [ 'gutenbergFSESettings', siteId ];
+
+export type SuccessResponse< T > = {
+	body: T;
+	headers: Record< string, string >;
+	status: 200 | 204;
+};
+
+export type GutenbergFSESettings = {
+	is_core_fse_eligible: boolean;
+};
+
+export const useGutenbergFSESettings = (
```

Comment:
`client/state` is not a good location for the React Query hooks. We put these into `client/data` subfolders.